internal/servicedeployer/compose.go

```diff
@@ -42,6 +42,8 @@ type DockerComposeServiceDeployerOptions struct {
 type dockerComposeDeployedService struct {
 	ctxt ServiceContext
 
+	shutdownTimeout string // default shutdown timeout 10 seconds
```

Comment:
Is this intended to be internal only? Maybe we can expose it to the user through some test setting.

Reply:
I'd prefer to keep it internal for now, to ensure that in the terraform service type it is set a high timeout. Currently, for that case is 300 secs, and all the other scenarios that flag (-t) would not be set, so it is 10 seconds according to the documentation.